extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -340,7 +341,7 @@ void ConsumeWindowsEventLog::onTrigger(const std::shared_ptr<core::ProcessContex
 
   size_t eventCount = 0;
   const TimeDiff timeDiff;
-  utils::ScopeGuard timeGuard([&]() {
+  gsl::finally timeGuard([&]() {
```

Comment:
same as [this](https://github.com/apache/nifi-minifi-cpp/pull/849#discussion_r460123115)

Reply:
Missed this one. :S